.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Whoops! Added this template by mistake, but still useful so I'm going to leave it.